modules/users/manifests/katstevens.pp

```diff
@@ -0,0 +1,7 @@
+# Creates the katstevens user
+class users::katstevens{ govuk_user { 'katstevens':
+    fullname => 'Kat Stevens',
```

Comment:
I think this needs to match your email address, so `katherinestevens`

Reply:
Whoops, on the line above

Reply:
Yes:
```
Failures:

  1) users::katstevens should have a username of the correct form
     Failure/Error: expect(uname_from_full).to eq(uname_from_email), "name in fullname(#{user[:fullname]}) must match name in email(#{user[:email]})"
       name in fullname(Kat Stevens) must match name in email(katherine.stevens@digital.cabinet-office.gov.uk)
     # ./modules/users/spec/classes/users_spec.rb:39:in `block (3 levels) in <top (required)>'
```

Reply:
Yup - see the error message in the build:
```
Failures:

  1) users::katstevens should have a username of the correct form
     Failure/Error: expect(uname_from_full).to eq(uname_from_email), "name in fullname(#{user[:fullname]}) must match name in email(#{user[:email]})"
       name in fullname(Kat Stevens) must match name in email(katherine.stevens@digital.cabinet-office.gov.uk)
     # ./modules/users/spec/classes/users_spec.rb:39:in `block (3 levels) in <top (required)>'
```

Reply:
Thanks all - pushed a fix. I've changed to `katherine` everywhere for consistency 😸